webpay/controllers/front/oneclickinscriptionvalidate.php

```diff
@@ -88,7 +89,7 @@ protected function setErrorTemplate(array $result)
         $error = isset($result['error']) ? $result['error'] : '';
         $detail = isset($result['detail']) ? $result['detail'] : '';
 
-        (new LogHandler())->logError($error.' ('.$detail.')');
+        $this->logError($error.' ('.$detail.')');
```

Comment:
is 'this->logError()' ok?
should be 'this->logger->logError()'?

Reply:
the method is in the parent 'BaseModuleFrontController'